cmd/co/main.go

```diff
@@ -46,12 +46,15 @@ func init() {
 	whoCmd.Flags().StringSliceVarP(&ownerFilters, "owner", "o", nil, "filter results by owner")
 	whoCmd.Flags().BoolVarP(&showUnowned, "unowned", "u", false, "only show unowned files (can be combined with -o)")
 	whoCmd.Flags().BoolP("json", "j", false, "format output as json. output is Array<{path: string; owners: Array<string>}>.")
+	root.AddCommand(whoCmd)
 
 	whyCmd.Flags().BoolP("json", "j", false, "format output as json. output is {path: string; line: string; rule: string; owners: Array<string>}.")
 
-	root.AddCommand(whoCmd)
 	root.AddCommand(whyCmd)
 
+	statsCmd.Flags().BoolP("json", "j", false, "format output as json")
+	root.AddCommand(statsCmd)
+
```

Comment:
When everything is wrapped up, can we also be sure to add the help output to the readme?

Reply:
Done, good call.